vignettes/model_definition.Rmd

```diff
@@ -0,0 +1,129 @@
+
+$$X_{t, \le d} = \sum_{i=0}^d X_{t,i} $$
+
+Such that $X_t = X_{t, \le D}$ is the “final” number of reported cases on time $t$. Conversely, for $d < D$
+
+$$X_{t,>d} = \sum_{i = d+1} ^{D} X_{t,i}$$
+
+is the number of cases still missing after $d$ delays.
+We refer to $X_t$ to describe a random variable, $x_t$ for the corresponding observation, and $\hat{x}_t$ for an estimated/imputed value.
+We refer to the matrix of $x$ available at a given data release time $t^*$ , as the reporting triangle. Here, all $t+d>t^*$ have yet to be observed.
+
+|   | $d = 0$ | $d = 1$ | $d=2$ | $...$ | $d= D-1$ | $d= D$ |
+|-----------|-----------|-----------|-----------|-----------|-----------|-----------|
+| $t=1$ | $x_{1,0}$ | $x_{1,1}$ | $x_{1,2}$ | $...$ | $x_{1,D-1}$ | $x_{1, D}$ |
+| $t=2$ | $x_{2,0}$ | $x_{2,1}$ | $x_{2,2}$ | $...$ | $x_{2,D-1}$ | $x_{2, D}$ |
+| $t=3$ | $x_{3,0}$ | $x_{3,1}$ | $x_{3,2}$ | $...$ | $x_{3,D-1}$ | $x_{3, D}$ |
+| $...$ | $...$ | $...$ | $...$ | $...$ | $...$ | $...$ |
+| $t=t^*-1$ | $x_{t^*-1,0}$ | $x_{t^*-1,1}$ | $x_{t^*-1,,2}$ | $...$ | $x_{t^*-1,,D-1}$ | $x_{t^*-1,D}$ |
+| $t=t^*$ | $x_{t^*,0}$ | $x_{t^*,1}$ | $x_{t^*,2}$ | $...$ | $x_{t^*,D-1}$ | $x_{t^*, D}$ |
+
+### Point estimate of the delay distribution
+
+We use the entire reporting triangle to compute an empirical estimate of the delay distribution, $\pi(d)$, or the probability that a case at reference time $t$ appears in the dataset at time $t + d$. We will refer to the realized empirical estimate of the delay distribution from a reporting triangle as $\pi(d)$.
+The delay distribution, $\pi(d)$ can be estimated directly from the completed reporting matrix $X$
+
+$$
+{\pi}_d= \frac{\sum_{t=1}^{t=t^*} X_{t,d}}{\sum_{d=0}^{D} \sum_{t=1}^{t=t^*} X_{t,d}}
+$$
+
+In the special case when the time the estimate is made $t'$, is beyond the data release time $t^{*}$, such that $t' \ge t^* + D$, $\hat{\pi}_d$ can be computed directly by summing over all reference time points $t$ at each delay $d$.
+In the case where there are partial observations, in order to properly weight the denominator with the missing delays, we have to first impute the cases $\hat{x}_{t,d}$ for all instances where $t+d > t^*$. This amounts to computing the point nowcast from the partial reporting triangle.
+
+### Point nowcast from incomplete reporting matrix
+
+To do so, we start by defining $\theta_d$, which is the factor by which the cases on delay $d$ compare to the total cases through delay $d-1$, obtained from $N$ preceding rows of the triangle. In practice, $N \ge D$, with any $N > D$ representing the number of completed observations used to inform the estimate
+
+$$
+\hat{\theta}_d(t^*) = \frac{\sum_{i=1}^{N} x_{t^*-i+d, d}}{\sum_{d=1}^{d-1} \sum_{i=1}^{N} x_{t^*-i+d,d}}
```

Comment:
Notationally all the indexes here feel a bit comlex for what it is. I dont have any great thoughts on simplification though

Reply:
Yep this was where I was struggling a bit last week, its way more complex in math notation than in just looking at the code and the blocks of the matrix being summed over...

Reply:
I was wondering if you maybe redefine the sum?

Reply:
Can you expand what you mean here? e.g. define a sum across reference times for a particular delay $d$, and define separately a sum across delays and reference times

Reply:
yes precisely, I dont think it is critical though